doc/api/fs.md

```diff
@@ -371,6 +371,73 @@ If one or more `filehandle.read()` calls are made on a file handle and then a
 position till the end of the file. It doesn't always read from the beginning
 of the file.
 
+#### `filehandle.readStream([options])`
```

Comment:
I'd prefer `readableNodeStream()` to better differentiate with the `readableWebStream()` used to return a `ReadableStream`.

Reply:
Note that this method returns a `fs.ReadStream`, which is a `stream.Readable`. The naming is indeed quite confusing, but I'm, unsure your suggestion really improves clarity: `readableNodeStream` is not really a thing, do we even refer to Node.js as Node anywhere in out API?
Here is not the right place to discuss this method, but maybe we should reconsider the `readableWebStream` method. I don't think it's standard in any way, and I suppose folks will be more inclined to use `fetch(fileHandle)` or `fetch(fileURLToPath)` if they want to use WHATWG `ReadableStream`.

Reply:
What about `createReadStream`?